fuse/src/main/java/com/github/kittinunf/fuse/core/Cache.kt

```diff
@@ -30,16 +30,20 @@ object CacheBuilder {
     }
 }
 
-fun <T : Any> Config<T>.build() = Cache(this)
+fun <T : Any> Config<T>.build(): Cache<T> = CacheImpl(this)
```

Comment:
The caller of `build()` should not need to know what specific implementation of `Cache` is used. So this was changed to just return the interface type.